HOLD FOR RELEASE - 73619: Regex Feature (#1049)
replicatedhq/replicated-docs: docs/vendor/config-screen-about.md

# About the Configuration Screen

> Applications can include a configuration screen in the Replicated admin console to collect required or optional values from your users that are used to run your application. You can configure some fields, such as passwords and email addresses, with regular expressions to help customers validate their input immediately. For more information about how to add custom fields to the configuration screen, see [Creating and Editing Configuration Fields](admin-console-customize-config-screen).

Reviewer: ```suggestion
Applications can include a configuration screen in the Replicated admin console to collect required or optional values from your users that are used to run your application. You can use regular expressions to validate user input for some fields, such as passwords and email addresses. For more information about how to add custom fields to the configuration screen, see [Creating and Editing Configuration Fields](admin-console-customize-config-screen).
```
